app/src/main/kotlin/cz/covid19cz/erouska/ui/dashboard/DashboardVM.kt

```diff
@@ -17,15 +26,54 @@ class DashboardVM(private val prefs: SharedPrefsRepository) : BaseVM() {
 
         // TODO Check last exposure
         // If last exposure occured in less than 14 days -> publish DashboardCommandEvent.Command.RECENT_EXPOSURE
+    }
 
+    @OnLifecycleEvent(Lifecycle.Event.ON_RESUME)
+    fun onResume() {
+        viewModelScope.launch {
+            kotlin.runCatching {
```

Comment:
NIT: Tohle neznám, nebyl by hezčí normální try - catch?

Reply:
výhodou je, že to onSuccess a onFailure volá na main threadu